.env

```diff
@@ -0,0 +1,2 @@
+FIRMWARE_VERSION=2021.11.22.0-1
+DIAGNOSTICS_VERSION=371d0c6
```

Comment:
Why do we need a .env file and a settings.ini? This seems to defeat the purpose of the PR of having all versions in a single file

Reply:
@shawaj - The title has become misleading on this issue. We're no longer using `.env` file. Versions are fetched from `settings.ini`.

Reply:
@kashifpk sure but there is still a .env file being added here, seemingly unnecessarily?